Docker/README.md

```diff
@@ -213,7 +213,7 @@ Remember not pass the `CRON_MIN` environment variable to your Docker run, to avo
 Example on Debian / Ubuntu: Create `/etc/cron.d/FreshRSS` with:
 
 ```
-7,37 * * * * root docker exec --user www-data -it freshrss php ./app/actualize_script.php > /tmp/FreshRSS.log 2>&1
+7,37 * * * * root docker exec --user www-data -i freshrss php ./app/actualize_script.php > /tmp/FreshRSS.log 2>&1
```

Comment:
```suggestion
7,37 * * * * root docker exec --user www-data freshrss php ./app/actualize_script.php > /tmp/FreshRSS.log 2>&1
```

I believe we can remove the `-i` as well

Reply:
True, nobody is waiting in front of a cron routine :)